Cargo.toml

```diff
@@ -21,14 +21,20 @@ use-serde = ["serde", "bitcoin_hashes/serde", "secp256k1/serde"]
 secp-lowmemory = ["secp256k1/lowmemory"]
 secp-recovery = ["secp256k1/recovery"]
 
+# either std or no-std must be selected
```

Comment:
This comment is misleading, should be "at least one of `std`, `no-std` must be enabled".
Since you prefer the name `no-std`, I'd suggest also adding clarification: "`no-std` doesn't disable `std` - you need to turn off `std` for that. Instead `no-std` enables additional features required for this crate to be usable without `std`. As a result, both can be enabled without conflict."

Reply:
Done